config/routes.rb

```diff
@@ -5,6 +5,7 @@
     resources :book, only: [:index, :show]
     resources :rent, only: [:index, :create]
     resources :book_suggestion, only: [:create]
+    get 'book/isbn', to: 'book#isbn'
```

Comment:
You are using BookController so do not by shy and use `resources :book`
Tips: You will need `collection`